samples/kubernetes/end2end/README.md

```diff
@@ -0,0 +1,31 @@
+## Monitoring WebLogic Server with Grafana Dashboard
+This is an end-to-end sample that walks through the steps to setup monitoring for WebLogic domains using Prometheus and Grafana from scratch. In the end you'll have Prometheus, Grafana, WLS domains installed, configured and running. And runtime metrics of WebLogic servers will be visualized in a Grafana dashboard.
+
+Firstly, let's check [what's in the WebLogic Dashboard](docs/dashboard.md).
+
+Before we go into the detail steps, see the diagram below for the overall architecture containing the basic components we'll deploy to Kubernetes cluster.
+
+![architecture](docs/images/architecture.png)
+
+Here's how the WebLogic metrics come all the way from WebLogic server and eventually displayed in the dashboard:
+- WebLogic servers exposes their runtime data via REST api.
+- The exporter running on each WebLogic server achieves WebLogic data via calling REST api, translates to the Prometheus metrics format and exposes in a HTTP endpoint.
+- Prometheus server is responsible to periodically scrape the metrics from the endpoints and store to its time series database.
```

Comment:
Prometheus server is responsible to periodically -> Prometheus server is responsible for periodically